src/fmeval/model_runners/extractors/jumpstart_extractor.py

```diff
@@ -33,19 +34,29 @@ class JumpStartExtractor(Extractor):
     """
 
     def __init__(
-        self, jumpstart_model_id: str, jumpstart_model_version: str, sagemaker_session: Optional[Session] = None
+        self,
+        jumpstart_model_id: str,
+        jumpstart_model_version: str,
+        is_embedding_model: Optional[bool] = False,
+        sagemaker_session: Optional[Session] = None,
     ):
         """
         Initializes  JumpStartExtractor for the given model and version.
         This extractor does not support batching at this time.
 
         :param jumpstart_model_id: The model id of the JumpStart Model
         :param jumpstart_model_id: The model version of the JumpStart Model
+        :param is_embedding_model: Whether this model is an embedding model or not
         :param sagemaker_session: Optional. An object of SageMaker session
         """
         self._model_id = jumpstart_model_id
         self._model_version = jumpstart_model_version
         self._sagemaker_session = sagemaker_session if sagemaker_session else get_sagemaker_session()
+        self._is_embedding_model = is_embedding_model
+
+        if self._is_embedding_model:
+            self._embedding_compiler = jmespath.compile(EMBEDDING_JMESPATH_EXPRESSION)
+            return
```

Comment:
Can we guarantee that all Jumpstart embedding models output *only* embeddings and nothing else? By returning early, we're saying that we know for certain that we won't need to set `self._output_jmespath_compiler` and `self._log_prob_compiler`.

Reply:
Per offline discussion, the answer is `yes` for now